CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ FetchContent_Declare(googletest
     URL "https://github.com/google/googletest/archive/refs/tags/v1.14.0.tar.gz")
 FetchContent_Declare(ios-cmake
     GIT_REPOSITORY https://github.com/leetal/ios-cmake.git
-    GIT_TAG 4.4.1)
+    GIT_TAG a52c591bc9056f3ae20a461d16fdd73ab4325be7)
```

Comment:
Is this change needed because we rely on this toolchain for visionOS as well? Is there a release tag that includes visionOS support, or we have to just grab a specific commit?

Reply:
They recently released a new version which Im going to bump in another PR just to make sure other builds are still working as they should.